index.html

```diff
@@ -15,7 +15,7 @@
 <body>
   <h1 id="header">Welcome to Ian's Liverpool FC Fan Club</h1>
   <h2>Add New Player</h2>
-  <form id="new-player-form">
+  <form id="new-player-form" method ='POST'>
```

Comment:
Since you're handling the form submit in Javascript, it's not necessary to add the method here.